tests/matrix_free/matrix_vector_hessians_cells.cc

```diff
@@ -101,7 +102,7 @@ test_hessians(const dealii::FE_Poly<dim>                    &fe,
   matrix_free.template loop<LinearAlgebra::distributed::Vector<double>,
                             LinearAlgebra::distributed::Vector<double>>(
     [&](
-      const auto &matrix_free, auto &dst, const auto &src, const auto &range) {
+      const auto &matrix_free, auto &dst, const auto &src, [[maybe_unused]]const auto &range) {
```

Comment:
This does not look right. You're not removing a `(void)range;` statement in the body of this lambda function.